client/my-sites/checkout/checkout-thank-you/custom-domain-purchase-detail.jsx

```diff
@@ -34,17 +34,18 @@ const CustomDomainPurchaseDetail = ( {
 				description={
 					onlyBlogDomain
 						? translate(
-								'Your plan includes a free .blog domain, which gives your site a more professional, branded feel.'
+								'Your plan includes a free .blog domain for one year, which gives your site a more professional, branded feel.'
```

Comment:
:ok: This change is safe, we'll keep the existing 17 translations.

Reply:
Not sure what this means. The text change is meant to clarify policy so I would expect new translations to be necessary. /cc @akirk